packages/runner/src/lib/event-manager.js

```diff
@@ -296,6 +296,10 @@ const eventManager = {
       })
     })
 
+    Cypress.on('test:before:run:async', () => {
+      Cypress.backend('reset:xhr:server')
+    })
```

Comment:
This should go in `packages/driver/src/cy/commands/xhr.coffee`

Refer to these files as reference:
- packages/driver/src/cy/commands/cookies.coffee
- packages/driver/src/cy/commands/window.coffee